### ISSUE TITLE
Sanity check fails for Clang 16.x when building the nvptx target

### ISSUE THREAD

**Issue by damianam** [easybuilders/easybuild-easyblocks#3040]
Clang 16 does not generate the `libomptarget-new-nvptx` files when building for the nvptx target, just `libomptarget-nvptx`:

```
$ ls *omptarget*nvptx*
libomptarget-nvptx-sm_70.bc
```

Logically the sanity check fails. I am unsure if this was introduced in 16.x or somewhen before (14.x, 15.x).

Related to/introduced in: https://github.com/easybuilders/easybuild-easyblocks/issues/2782

**Comment by boegel** [comment]
Same problem with Clang 15.0.5:
```
  >> file 'lib/libomptarget.so' found: OK
  >> file 'lib/libomptarget.rtl.x86_64.so' found: OK
  >> file 'lib/libomptarget.rtl.cuda.so' found: OK
  >> file 'lib/libomptarget-nvptx-sm_70.bc' found: OK
  >> file 'lib/libomptarget-new-nvptx-sm_70.bc' found: FAILED
```

**Comment by boegel** [comment]
fixed in #3059
